src/views/CardStack/CardStack.js

```diff
@@ -38,6 +38,14 @@ import TransitionConfigs from './TransitionConfigs';
 
 const emptyFunction = () => {};
 
```

Comment:
why not simply:

```js
const EaseInOut = Easing.inOut(Easing.ease);
...
easing: EaseInOut,
```

There is no need to over-optimize this

Reply:
Sure, just did what `Animated` does under the hood (see source in the links above).
I am happy to change that, however.

Also, I am thinking about another change to types, will post the idea when I get to desktop but it probably deserves another PR